source/releasenotes/about.rst

```diff
@@ -17,11 +17,11 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is the initial 4.18 LTS release. It has over 300 fixes
-and features since the 4.17.2.0 release.
+Apache CloudStack |release| is a 4.18 LTS minor release with about xxx fixes
```

Comment:
@weizhouapache do you want to fix this? by saying more than 100/200 fixes?

Reply:
updated to "196 fixes"